packages/server/lib/util/print-run.ts

```diff
@@ -168,16 +168,16 @@ function formatNodeVersion ({ resolvedNodeVersion, resolvedNodePath }: Pick<Cfg,
   return
 }
 
-function formatRecordParams (runUrl?: string, parallel?: boolean, group?: string, tag?: string) {
+function formatRecordParams (runUrl?: string, parallel?: boolean, group?: string, tag?: string, autoCancelAfterFailures?: number | false) {
   if (runUrl) {
-    return `Tag: ${tag || 'false'}, Group: ${group || 'false'}, Parallel: ${Boolean(parallel)}`
+    return `Tag: ${tag || 'false'}, Group: ${group || 'false'}, Parallel: ${Boolean(parallel)}, Auto Cancel After Failures: ${autoCancelAfterFailures || 'false'}`
```

Comment:
```suggestion
    return `Tag: ${tag || 'false'}, Group: ${group || 'false'}, Parallel: ${Boolean(parallel)}, Auto Cancel After Failures: ${autoCancelAfterFailures}`
```

Reply:
Should this fallback to false here? It would indicate this value is false when really it would be the global configuration right?

Reply:
Also thinking about this, if only some plans support auto cancellation, should we only conditionally show this in the run console?